src/healthcheck/HealthChecker.ts

```diff
@@ -86,20 +88,27 @@ class HealthChecker {
     // Force this to be an instance function so that it can access `this` fields
     this.onShutdownRequest = () => {
       this.shutdownRequested = true;
-      for (let plugin in this.shutdownPlugins) {
-        if (this.shutdownPlugins[plugin] != undefined) {
-          this.shutdownPlugins[plugin].runCheck()
-        }
-      }
+      this.shutdownPlugins.map((check) => {
+        check.runCheck()
+      })
+      //for (let plugin in this.shutdownPlugins) {
```

Comment:
nit: we can remove this commented code as the above works

Reply:
Removed.